.github/workflows/build_nugets.yml

```diff
@@ -0,0 +1,98 @@
+name: Windows Nuget
+
+on:
+  push:
+    branches:
+      - master
+      - 'releases/**'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build_nuget_windows:
+    name: nuget.${{ matrix.toolset }}.${{ matrix.arch_config.arch }}
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: ["windows-2019"]
+        toolset: ["v141", "v142"]
+        arch_config:
+        - { arch: x64, generator_arch: x64}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - uses: ilammy/msvc-dev-cmd@v1
+      - name: Configure
+        run: >
+          cmake -S . -B build
+          -G "Visual Studio 16 2019" -A ${{ matrix.arch_config.generator_arch }}
+          -T ${{ matrix.toolset }}
+          -DNATIVE_NUGET_PLATFORM_TAG="${{ matrix.arch_config.arch }}"
+          -DBUILD_FLATBUFFERS=Off
+          -DBUILD_TESTING=Off
+          -DRAPIDJSON_SYS_DEP=Off
+          -DFMT_SYS_DEP=Off
+          -DSPDLOG_SYS_DEP=Off
+          -DVW_ZLIB_SYS_DEP=Off
+          -DVW_BOOST_MATH_SYS_DEP=Off
+          -DDO_NOT_BUILD_VW_C_WRAPPER=On
+          -DVW_INSTALL=On
+      - name: Build and install
+        run: |
+          cmake --build build --config Debug -t vw_io vw vw-bin allreduce spanning_tree
+          cmake --build build --config Release -t vw_io vw vw-bin allreduce spanning_tree
+
+          # Install debug first so that release overwrites the exe and nothing else
+          cmake --install build --prefix ./nuget_staging --config Debug
```

Comment:
so the executable we get is only release right?

Reply:
why do we need the debug build?

Reply:
> so the executable we get is only release right?
 Correct

> why do we need the debug build?
Since VS is multi-config you can build both in the same solution. It also seems to sorta be a convention among nuget packages to supply both release and debug bits